libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -40,8 +40,11 @@ You can specify the following options in the `setup.dashboards` section of the
 ==== `setup.dashboards.enabled`
 
 If this option is set to true, {beatname_uc} loads the sample Kibana dashboards
-automatically on startup. If no other options are set, the dashboard are loaded
```

Comment:
Why the sentence which start with "If no other options are set..." is removed? AFAIK it is still possible to override it using `setup.dashboards.directory`.

Reply:
@kvch I think this was my fault! When I suggested an edit to @loekvangool, I inadvertently deleted that line when I copy/pasted the original text. I didn't notice the line was missing because the sentence read correctly without the deleted line. My edit *should* have said:

```
If this option is set to true, {beatname_uc} loads the sample Kibana dashboards
automatically on startup. If no other options are set, the dashboard are loaded
from the local `kibana` directory in the home path of the {beatname_uc} installation.
```

Sorry about the confusion.